### ISSUE TITLE
Order add, remove, cancel -> Log

### ISSUE THREAD

**Issue by kilrau** [ExchangeUnion/xud#649]
add a log line every time an order is added or removed or cancelled

**Comment by offerm** [comment]
or matched. or reduced. 

In general add debug entry everytime there is an order change.

**Comment by sangaman** [comment]
We actually have logging for when individual orders are added and removed/reduced/canceled, but not much for when orders are matched. I'm working on that.

**Comment by kilrau** [comment]
and when order invalidation message was sent, received, processed (https://github.com/ExchangeUnion/xud/pull/626#discussion_r232150920)

**Comment by sangaman** [comment]
We already have that for the most part re: order invalidation. We log [when the packet is received and processed](https://github.com/ExchangeUnion/xud/blob/3868681741bada5ac39166152f14530ee3873f3a/lib/p2p/Pool.ts#L490) and when an [order is removed or reduced for any reason](https://github.com/ExchangeUnion/xud/blob/3868681741bada5ac39166152f14530ee3873f3a/lib/orderbook/TradingPair.ts#L206).

We also have trace level logging for every time a packet is received - I can add trace level logging every time we send a packet as well, but I think it would get too cluttered if we go with a higher log level than that. Order invalidation packets are common and we'd have a line for each peer every time an own order is matched or removed for any reason.

**Comment by offerm** [comment]
I think we need a debug pring every time we are making a change.

So, sending invalidation is not a change. reducing the quantity as a result of invalidation is a change.

**Comment by sangaman** [comment]
That makes sense, but we do log the result of the invalidation currently.

I'm also thinking we can log a single debug line when we broadcast an order invalidation packet, which only takes a single line and still clearly shows that the packet was sent.

**Comment by sangaman** [comment]
I am thinking it's probably worth changing the log message when we receive an order invalidation packet though so it's more clear, I'll open a PR that does that along with the other changes in this issue soon.

**Comment by offerm** [comment]
@sangaman, be aware that the reason I haven't see the invalidation log message is that I looked at the taker's log file. The taker is excluded from the invalidation message.

**Comment by sangaman** [comment]
@offerm Yes that's true, that's intentional though as we don't send an order invalidation to the peer that matched our order (because they will have already removed it from the order book during matching). The issue is that we don't have logging when orders are removed due to matching, but I addressed that in #662.
